www/_data/toc/en_8-x-src.yml

```diff
@@ -0,0 +1,76 @@
+-
+    name: Introduction
+    children:
+        - url: guide/overview/index.html
+-
+    name: Create apps
+    children:
+        - url: guide/cli/index.html
+        - url: guide/cli/template.html
+        - url: guide/support/index.html
+        -
+            name: Develop for platforms
+            children:
+                - url: guide/platforms/android/index.html
+                - url: guide/platforms/blackberry10/home.html
```

Comment:
we should probably remove the deprecated platforms here, should we?

Reply:
If you deprecate it, make sure to add it to `redirects.yml`.